src/sagemaker/jumpstart/curated_hub/curated_hub.py

```diff
@@ -143,18 +151,22 @@ def describe(self) -> DescribeHubResponse:
 
         return hub_description
 
-    def list_models(self, **kwargs) -> Dict[str, Any]:
-        """Lists the models in this Curated Hub
+    
+    def list_models(self, clear_cache: bool = True, **kwargs) -> List[Dict[str, Any]]:
```

Comment:
is `clear_cache` used anywhere else in this module? 

personally i'm not a fan of this, i'd prefer if you use `lru_cache`, it's much simpler imo